src/instance.jl

```diff
@@ -224,6 +224,9 @@ MOI.canget(m::AbstractInstance, ::Union{MOI.NumberOfVariables,
                                         MOI.ObjectiveFunction,
                                         MOI.ObjectiveSense}) = true
 
+MOI.canset(m::AbstractInstance, ::Union{MOI.ObjectiveFunction,
```

Comment:
Not needed anymore, it has been added in https://github.com/JuliaOpt/MathOptInterfaceUtilities.jl/pull/54